Userland/Libraries/LibGfx/BMPWriter.h

```diff
@@ -12,17 +12,22 @@ namespace Gfx {
 
 class Bitmap;
 
-class BMPWriter {
-public:
-    BMPWriter() = default;
-
+// This is not a nested struct to work around llvm.org/PR36032.
```

Comment:
that's a closed issue, what am I missing?

Reply:
Copy-paste fail, is supposed to say https://llvm.org/pr36684